frontend/app/src/people/widgetViews/userTicketsView.tsx

```diff
@@ -0,0 +1,165 @@
+        const value = { ...body };
+        return (
+            bountyHeaderFilter(data.checkboxIdToSelectedMap, value?.paid, !!value?.assignee) &&
+            bountyHeaderLanguageFilter(value?.codingLanguage, {})
+        );
+    });
+
+    async function getUserTickets() {
+        setIsLoading(true);
+
+        const tickets = await main.getPersonAssignedWanteds({}, personPubkey);
+
+        setUserTickets(tickets);
+
+        setIsLoading(false);
+    }
+
+    function onPanelClick(i) {
+        history.push({
+            pathname: `${url}/${i}`
+        });
+    }
+
+    const confirmDelete = async () => {
+        try {
+            if (deletePayload) {
+                await deleteTicket(deletePayload);
+            }
+        } catch (error) {
+            console.log(error);
+        }
+        closeModal();
+    };
+
+    const deleteTicket = async (payload: any) => {
+        const info = ui.meInfo as any;
+        const URL = info.url.startsWith('http') ? info.url : `https://${info.url}`;
+        try {
+            await fetch(`${URL}/delete_ticket`, {
+                method: 'POST',
+                body: JSON.stringify(payload),
+                headers: {
+                    'x-jwt': info.jwt,
+                    'Content-Type': 'application/json'
+                }
+            });
+        } catch (error) {
+            console.log(error);
+        }
+    };
+
+    useEffect(() => {
+        getUserTickets();
+    }, [])
+
+    const listItems =
+        activeList && activeList.length ? (
+            activeList.slice(0, currentItems).map((item, i) => {
+                const { person, body } = item;
+
+                // if this person has entries for this widget
+                return (
+                    <Panel
+                        isMobile={isMobile}
+                        key={person?.owner_pubkey + i + body?.created}
+                    >
+                        <WantedView
+                            colors={color}
+                            showName
+                            onPanelClick={() => {
+                                if (onPanelClick) onPanelClick(i);
+                            }}
+                            person={person}
+                            showModal={showModal}
+                            setDeletePayload={setDeletePayload}
+                            fromBountyPage={false}
+                            {...body}
+                            show={true}
+                        />
+                    </Panel>
+                );
+            })
+        ) : (
+            <NoResults loading={loading} />
+        );
+
+    return (
+        <div data-testid="test">
+            <Container>
+                {/* <BrowserRouter> */}
+                <Switch>
+                    <Route path={`${path}/:wantedId`}>
+                        <BountyModal basePath={url} />
+                    </Route>
+                </Switch>
+                {/* </BrowserRouter> */}
```

Comment:
After doing this the test passed for me
```suggestion
                <BrowserRouter>
                <Switch>
                    <Route path={`${path}/:wantedId`}>
                        <BountyModal basePath={url} />
                    </Route>
                </Switch>
                </BrowserRouter>
```

Reply:
Using BrowserRouter makes the test pass, but it breaks the modal functionality, I’m finding a way of making it work seamlessly